.travis.yml

```diff
@@ -69,3 +69,19 @@ jobs:
       - ls wheelhouse/
       - python -m twine upload --skip-existing wheelhouse/anonlink*manylinux1_x86_64.whl
       - python -m twine upload --skip-existing wheelhouse/anonlink*.tar.gz
+    - name: "manylinux2010_x86_64"
```

Comment:
Should you also set the stage attribute?

Reply:
Nope, uses the last one.

![image](https://user-images.githubusercontent.com/855189/63417273-48389a00-c444-11e9-9d22-91bb78139ef1.png)

https://travis-ci.org/data61/anonlink/builds/574562100